.github/workflows/CI.yml

```
@@ -44,7 +44,7 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v2
-      - uses: julia-actions/julia-downgrade-compat@v1
+      - uses: julia-actions/julia-downgrade-compat@v2
         with:
           skip: LinearAlgebra,Markdown,Printf,Random,Statistics,Test
```

Unrelated to the PR, but I noticed in the logs that we forgot to set
```suggestion
          skip: LinearAlgebra,Markdown,Printf,Random,Statistics,Test
          julia_version: ${{ matrix.version }}
```

Apparently `julia_version` defaults to 1.10 which is incorrect since currently we run these downgrade tests only with 1.11 (`'1'`).